action.yml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Google LLC
```

Comment:
This is just a rename with a few changes. I'm not sure why GitHub is showing a full diff.